src/main/java/utils/Credentials.java

```diff
@@ -0,0 +1,34 @@
+package utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class Credentials {
+
+    public String username;
+    public String password;
+    private String path = "src/test/resources/userCredentials";
+    private static final Credentials instance = new Credentials();
+
+    private Credentials() {
+        FileInputStream fileInputStream;
```

Comment:
You need to close stream if it was opened, in catch fileInputStream.close() or try-with-resources